src/tests/krate/yanking.rs

```diff
@@ -220,3 +220,66 @@ async fn publish_after_yank_max_version() {
     let json = anon.show_crate("fyk_max").await;
     assert_eq!(json.krate.max_version, "2.0.0");
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn patch_version_yank_unyank() {
+    let (_, anon, _, token) = TestApp::full().with_token();
+
+    // Upload a new crate
+    let crate_to_publish = PublishBuilder::new("patchable", "1.0.0");
+    token.publish_crate(crate_to_publish).await.good();
+
+    // Check initial state
+    let json = anon.show_version("patchable", "1.0.0").await;
+    assert!(!json.version.yanked);
+    assert_eq!(json.version.yank_message, None);
+
+    // Yank with message
+    token
+        .update_yank_status("patchable", "1.0.0", Some(true), Some("Yanking reason"))
+        .await
+        .good();
```

Comment:
we might want to add assertions on the response JSON here

Reply:
Updated.